dashboard/client/src/pages/serve/ServeApplicationsListPage.tsx

```diff
@@ -78,18 +86,17 @@ export const ServeApplicationsListPage = () => {
   }
 
   return (
-    <div>
+    <div className={classes.root}>
       {serveDetails.http_options === undefined ? (
         <Alert className={classes.serveInstanceWarning} severity="warning">
           Serve not started. Please deploy a serve application first.
         </Alert>
       ) : (
         <React.Fragment>
-          <ServeSystemDetails
-            serveDetails={serveDetails}
+          <ServeSystemPreview
```

Comment:
> We can also remove having Serve applications be a collapsible section, and make the title of the page Applications instead (similar to System tab)

Serve application -> Application makes sense.

Let's keep the collapsable behavior for now since it doesn't hurt? I don't know if people might want to collapse it when the table is long and they just want to focus on logs and metrics.